tests/integrations/config/mod.rs

```diff
@@ -810,6 +810,7 @@ fn test_serde_custom_tikv_config() {
         initial_scan_pending_memory_quota: ReadableSize::kb(2),
         initial_scan_rate_limit: ReadableSize::mb(3),
         min_ts_interval: ReadableDuration::secs(2),
+        ..Default::default()
```

Comment:
Could you test `temp_file_memory_quota`?

Reply:
This config term cannot be tested because `#[serde(skip_serialization)]` will make the dump result inconsistent.